client/templates/reporting/draft-audit.njk

```diff
@@ -0,0 +1,13 @@
+{% extends "../layouts/default-no-nav.njk" %}
+
+{% block page_title %}{{ serviceName}} - {{ serviceTitle }}{% endblock %}
```

Comment:
we probably need to replace this `{{ serviceTitle }}` as it is not set on the controller.

Reply:
thinking about it, this page is just a redirect 😄